doc/menu_lua_api.txt

```diff
@@ -186,11 +186,11 @@ core.log(loglevel, line) (possible in async calls)
 ^ loglevel one of "error", "action", "info", "verbose"
 
 Settings:
-core.setting_set(name, value)
-core.setting_get(name) -> string or nil (possible in async calls)
-core.setting_setbool(name, value)
-core.setting_getbool(name) -> bool or nil (possible in async calls)
-core.setting_save() -> nil, save all settings to config file
+core.settings:set(name, value)
+core.settings:get(name) -> string or nil (possible in async calls)
+core.settings:setbool(name, value)
+core.settings:getbool(name) -> bool or nil (possible in async calls)
```

Comment:
`get_bool` and `set_bool`
It's also incomplete and easily forgotten in changes. Perhaps we should simply refer to the `lua_api.txt` docs for the settings object.